File: Fundamentals with JS/T10 - Functions/lecture/04. Orders.js
function orders(item, quantity) {
    let coffeePrice = 1.50
    let waterPrice = 1.00;
    let cokePrice = 1.40;
    let snacksPrice = 2.00;

    switch (item) {
        case "coffee":
            return (coffeePrice * quantity).toFixed(2);
        case "water":
            return (waterPrice * quantity).toFixed(2);
        case "coke":
            return (cokePrice * quantity).toFixed(2);
        case "snacks":
            return (snacksPrice * quantity).toFixed(2);
    }
}
console.log(orders("water", 5))
console.log(orders("coffee", 2))

File: Fundamentals with Python/T09 - Functions/lecture/04. Orders.py
product = input()
quantity = int(input())

def calculate_order(product, quantity):
    order = 0
    if product == "coffee":
        order = 1.5 * quantity
    elif product == "coke":
        order = 1.4 * quantity
    elif product == "water":
        order = 1 * quantity
    elif product == "snacks":
        order = 2 * quantity
    return f'{order:.2f}'

print(calculate_order(product, quantity))